Task_TheGridProblem/index.html

```diff
@@ -1,14 +1,27 @@
-<!DOCTYPE html>
-<html>
-  <head>
-    <title>Grid Problem</title>
-    <link rel="stylesheet" href="index.css" />
-  </head>
-
-  <body style="margin: 0; padding: 0">
-    <div top="0">
-      <a href="../index.html">Go Back</a>
-    </div>
-    <!-- Start coding from the lines below for GRID PROBLEM TASK; DO NOT TOUCH THE CODE ABOVE-->
-  </body>
-</html>
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Grid Problem</title>
+    <link rel="stylesheet" href="index.css" />
+  </head>
+
+  <body style="margin: 0; padding: 0">

```

Comment:
There is an extra space in the class attribute value for this div element. Please remove the space.